fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -124,6 +124,9 @@ bard__default_asyncAfter=never
 # Flag to turn on case sensitive keys in keyvalue store
 bard__case_sensitive_keys_enabled = false
 
+# Default value for Druid limit on uncovered interval
```

Comment:
This is obviously the default because it's in this file. Instead, the comment should describe what the parameter is for, what it controls, and what this default value means or does.